packages/jest-transform/src/__tests__/should_instrument.test.js

```diff
@@ -6,20 +6,15 @@
  *
  */
 
-import {normalize} from 'jest-config';
 import shouldInstrument from '../shouldInstrument';
+import {makeGlobalConfig} from '../../../../TestUtils';
 
 describe('shouldInstrument', () => {
   const defaultFilename = 'source_file.test.js';
   const defaultOptions = {
     collectCoverage: true,
   };
-  const defaultConfig = normalize(
-    {
-      rootDir: '/',
-    },
-    {},
-  ).options;
+  const defaultConfig = makeGlobalConfig({rootDir: '/'});
```

Comment:
I had completely forgotten about this util 😅 Dropping the `jest-config` dep means this package is ready to be written in TS, which is nice